src/app.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import router from './routes/index';
+const app = express();
+
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.json()); 
+app.use(router);
+
+app.use('/',(req, res) => {
+    res.status(200).json({ success: true, message: "You are using Barefoot nomad app." })
+  });
```

Comment:
No end point are allowed to be in defined in `src\app.js` file. They will be defined in the `routers\index.js` file

Reply:
let me do it